packages/test-utils/src/wrapper.js

```diff
@@ -582,3 +587,23 @@ export default class Wrapper implements BaseWrapper {
     this.element.dispatchEvent(event)
   }
 }
+
+export const enableAutoDestroy = (hook: Function) => {
+  if (Wrapper._instances) {
+    throwError('enableAutoDestroy cannot be called more than once')
+  }
+
+  Wrapper._instances = []
+
+  hook(() => {
+    const { _instances } = Wrapper
+    if (!_instances) {
+      return
+    }
+
+    while (_instances.length > 0) {
```

Comment:
**question:** Would a `forEach` and then setting `_instances = [];` be faster? Since these arrays aren't structured like stacks I'm not sure if popping is the best way to iterate 🤔...

Reply:
I have no idea what happened here. Probably being too tired when I thought this would  be a good idea. :sleeping: